examples/mongodbatlas_encryption_at_rest_private_endpoint/azure/README.md

```diff
@@ -25,20 +37,22 @@ This example shows how to configure encryption at rest using Azure with customer
 - `azure_region_name`: Region in which the Encryption At Rest private endpoint is located
 
 
-NOTE: The Azure application (associated to `azure_client_id`) must have at least a Key Vault Contributor role assigned in the corresponding Key Vault.
+**NOTE**: The Azure application (associated to `azure_client_id`) must have the following permissions associated to the Azure Key Vault (`azure_key_vault_name`):
+- GET (Key Management Operation), ENCRYPT (Cryptographic Operation) and DECRYPT (Cryptographic Operation) policy permissions.
+- A `Key Vault Reader` role.
```

Comment:
Pending confirmation from API team that this role is effectively needed, current placing as it was needed to achieve a successful execution.

Reply:
Confirmed that Key Vault Reader role is needed.